log_weasel.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency("mocha")
   s.add_development_dependency("no_soup_for_you")
 
-  s.add_dependency("activesupport", "~> 4.1.1")
+  # s.add_dependency("activesupport", "~> 4.1.1")
+  s.add_dependency("activesupport")
```

Comment:
Igual con esto no la sacamos, sólo le sacamos la restricción de versión y usamos la última que funcione. Qué versión le queda al final? Creo que está bueno ser específicos en cuanto a versiones de dependencias de **gemas**, se puede ser más flexible con dependencias de aplicaciones (que usan gemas)

Reply:
OK, check it now